examples/python/transfer_l2_usdc.py

```diff
@@ -0,0 +1,136 @@
+import asyncio
+import logging
+import os
+
+from starknet_py.contract import Contract
+from starknet_py.net.account.account import Account
+
+from utils import (
+    get_account,
+    get_paradex_account_address,
+    get_paradex_config,
+    get_proxy_config,
+    get_random_max_fee,
+    hex_to_int,
+)
+
+# Transfer USDC from old Paradex account to new Paradex account
+async def paraclear_transfer(config: dict, old_account: Account, new_account: Account):
+    paraclear_address = config["paraclear_address"]
+    paraclear_decimals = config["paraclear_decimals"]
```

Comment:
just to understand: this decimals corresponds to USDC balance, right? if yes, why not retrieving it from USDC contract instead? current manual method is error-prone